maps/_DeepForest/map/_Deep_Forest.dmm

```diff
@@ -15418,7 +15418,7 @@
 "keY" = (
 /obj/structure/closet/crate/secure/plasma,
 /obj/item/ammo_magazine/ammobox/shotgun/stunshells,
-/obj/item/ammo_casing/shotgun/payload/prespawned,
```

Comment:
This still isnt fixed its commited out rather then proper removed

Reply:
fixed